packages/endpoints-overlay/src/index.tsx

```diff
@@ -80,22 +80,27 @@ function DefaultMapMarkerIcon({
   location,
   type
 }: UserLocationAndType): ReactElement {
+  const pixels = 20;
   let inner;
   switch (type) {
-    case "to":
+    case "to": {
+      // The fa-solid's "location-dot" icon used here has a width-height ratio of 3/4,
+      // so the desired width for the outline/"stacked" element is 4/3 of the "apparent" width.
+      const toPixels = pixels * 1.3;
```

Comment:
<img width="62" alt="Screen Shot 2022-10-20 at 2 26 49 PM" src="https://user-images.githubusercontent.com/86619099/197028685-2639c017-7f05-4976-9eed-e8be6b3763f5.png">
I think this is causing some strange behavior in safari

Reply:
Safari doesn't round CSS px units like the other browsers! Fixed in https://github.com/opentripplanner/otp-ui/pull/463/commits/e52e9233d55d2480c62822f7328fa912a0114728.